xen-hvm-stubdom-legacy.spec.in

```diff
@@ -0,0 +1,291 @@
+Patch616: patch-hvmpt02-disable-msix-caps.patch
+Patch617: patch-hvmpt03-passthrough-log.patch
+Patch618: patch-hvmpt04-minios-nomask-bar-addrs.patch
+Patch619: patch-hvmpt05-hide-pio-bars.patch
+Patch620: patch-hvmpt06-fix-msix.patch
+Patch621: patch-libxc-fix-xc_gntshr_munmap-semantic.patch
+Patch622: patch-minios-ignore-close-0.patch
+Patch624: patch-qemu-docs-utf8.patch
+Patch625: patch-minios-remove-net-device-instead-of-preparing-for-re.patch
+Patch627: patch-qemu-keyboard-leds.patch
+Patch628: patch-python-Adjust-xc_physinfo-wrapper-for-updated-virt_c.patch
+Patch634: patch-python-do-not-report-handled-EAGAIN-error.patch
+
+# GCC7 fixes
+Patch706: patch-mini-os-link-to-libgcc.a-to-fix-build-with-gcc7.patch
+# GCC8 fixes
+Patch714: patch-tools-kdd-mute-spurious-gcc-warning.patch
+# GCC9 fixes
+Patch716: patch-python-fix-Wsign-compare-warnings.patch
+
+# vchan for stubdom:
+Patch801: patch-0100-minios-enhance-xenstore-available-for-stubdoms.patch
+Patch803: patch-0102-libvchan-remove-unnecessary-includes.patch
+Patch804: patch-0103-minios-implement-gntalloc-interface.patch
+Patch805: patch-0104-libxc-implement-gntshr-for-minios.patch
+Patch806: patch-0105-stubdom-make-libvchan-available-in-stubdom.patch
+
+# Support for Linux based stubdom
+#    patch-stubdom-linux-v3-0000-cover-letter.patch
+Patch900: patch-stubdom-linux-v3-0001-Document-ioemu-MiniOS-stubdomain-protocol.patch
+Patch901: patch-stubdom-linux-v3-0002-Document-ioemu-Linux-stubdomain-protocol.patch
+Patch902: patch-stubdom-linux-v3-0003-libxl-fix-qemu-trad-cmdline-for-no-sdl-vnc-case.patch
+Patch903: patch-stubdom-linux-v3-0004-libxl-Allow-running-qemu-xen-in-stubdomain.patch
+Patch904: patch-stubdom-linux-v3-0005-libxl-Handle-Linux-stubdomain-specific-QEMU-options.patch
+Patch905: patch-stubdom-linux-v3-0006-libxl-write-qemu-arguments-into-separate-xenstore-keys.patch
+Patch906: patch-stubdom-linux-v3-0007-libxl-create-vkb-device-only-for-guests-with-graphics-output.patch
+Patch907: patch-stubdom-linux-v3-0008-xl-add-stubdomain-related-options-to-xl-config-parser.patch
+Patch908: patch-stubdom-linux-v3-0009-tools-libvchan-notify-server-when-client-is-connected.patch
+Patch909: patch-stubdom-linux-v3-0010-libxl-typo-fix-in-comment.patch
+Patch910: patch-stubdom-linux-v3-0011-libxl-move-xswait-declaration-up-in-libxl_internal.h.patch
+Patch911: patch-stubdom-linux-v3-0012-libxl-use-vchan-for-QMP-access-with-Linux-stubdomain-libxl__ev_qmp_-version.patch
+Patch912: patch-stubdom-linux-v3-0013-libxl-use-vchan-for-QMP-access-with-Linux-stubdomain-non-async-version.patch
+Patch913: patch-stubdom-linux-v3-0014-libxl-add-save-restore-support-for-qemu-xen-in-stubdomain.patch
+Patch914: patch-stubdom-linux-v3-0015-tools-add-missing-libxenvchan-cflags.patch
+Patch915: patch-stubdom-linux-v3-0016-libxl-add-locking-for-libvchan-QMP-connection.patch
+# modified vs upstream
+Patch916: patch-stubdom-linux-v3-0017-libxl-require-qemu-in-dom0-even-if-stubdomain-is-in-use.patch
```

Comment:
Those are for toolstack side of stubdomain (and linux one, not legacy), not needed here.